tasks/redirects.js

```diff
@@ -0,0 +1,13 @@
+module.exports = function( grunt ) {
+
+var wp = require( "wordpress" );
+
+grunt.registerTask( "redirects", function() {
+	var config = grunt.config( "wordpress" ),
```

Comment:
We don't really do onevar anymore (https://github.com/jquery/contribute.jquery.org/pull/105)